external-crates/move/move-compiler/src/naming/translate.rs

```diff
@@ -1411,6 +1461,39 @@ fn exp_(context: &mut Context, e: E::Exp) -> N::Exp {
                 }
             }
         }
+        EE::Call(ma, false, tys_opt, rhs) if context.is_struct_reference(&ma) => {
+            context
+                .env
+                .check_feature(FeatureGate::PositionalFields, context.current_package, eloc);
```

Comment:
Oh no. I suppose we do need to check this here...

Reply:
Yea, sadly no way to do this before this :'(